opentelemetry-proto/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## vNext
 
+- Update proto definitions to v1.8.0.
```

Comment:
we can update the existing entry to v1.8.0 since there was never a release with 1.7